Tiltfile

```diff
@@ -107,12 +92,11 @@ else:
     context='.',
     entrypoint="/cluster-agent/cluster-agent -c /etc/kubetail/config.yaml",
     only=[
-      './crates/rgkl',
+      './crates',
       './proto',
-      './.tilt/cluster-agent'
     ],
     ignore=[
-      './crates/rgkl/target'
+      './crates/*/target'
```

Comment:
I think we need to add './crates/target' here too

Reply:
Fixed in 43b54f78ed3d6f6e9a23349384de5f02eab63ffa